src/utils/network/profile-helper.js

```diff
@@ -0,0 +1,101 @@
+import {
+  buildThing,
+  getStringNoLocale,
+  getThing,
+  getWebIdDataset,
+  removeStringNoLocale,
+  saveSolidDatasetAt,
+  setThing
+} from '@inrupt/solid-client';
+import { RDF_PREDICATES } from '../../constants';
+
+/**
+ * @typedef {import('@inrupt/solid-ui-react').SessionContext} Session
+ */
+
+/**
+ * A function fetches the user's profile information from their webId's profile
+ * card
+ *
+ * @function fetchProfileInfo
+ * @param {Session} session - Solid's Session Object {@link Session}
+ * @returns {Promise<object>} Object - The object containing the information related
+ * to the person on their profile card, the profile dataset, and the profile Thing
+ */
+export const fetchProfileInfo = async (session) => {
+  const profileDataset = await getWebIdDataset(session.info.webId);
+  const profileThing = getThing(profileDataset, session.info.webId);
+
+  const profileName = getStringNoLocale(profileThing, RDF_PREDICATES.profileName);
+  const organization = getStringNoLocale(profileThing, RDF_PREDICATES.organization);
```

Comment:
No, you had this relationship correct the first time. The person is a member of an organization. The issue was that the data you put for organization was incorrect.

I'd suggest you read the schema documentation more carefully. `Organization` is not text. It's a thing. These functions need to retrieve and restore things, not strings.

Reply:
I think we can build another Thing for `Organization` inside the profile card in the mean time.

I'm not too sure about which functions could be use to set Things inside of other Things at this moment.

Reply:
Think I got this working as intended. Organization would be created as a new Thing and be adjust for accordingly.

https://github.com/codeforpdx/PASS/assets/14917816/bd30f241-e306-42c2-ad2c-bd4624d090b0